mobile-app/app/screens/WalletNavigator/screens/components/OnboardingCarouselV2.tsx

```diff
@@ -0,0 +1,110 @@
+import ImageDLight from '@assets/images/onboarding/welcome-screen-d-light.png'
+import { View } from '@components/index'
+import { ThemedText } from '@components/themed'
+import { useThemeContext } from '@shared-contexts/ThemeProvider'
+import { getColor, tailwind } from '@tailwind'
+import { translate } from '@translations'
+
+interface CarouselImage {
+  imageDark: ImageSourcePropType
+  imageLight: ImageSourcePropType
+  title: string
+  subtitle: string
+}
+
+const slides: JSX.Element[] = [
+  <ImageSlide
+    imageDark={ImageA}
+    imageLight={ImageALight}
+    key={0}
+    subtitle='DeFiChain Wallet is fully non-custodial. Only you will have access to your fund.'
+    title='Take full control'
+  />,
+  <ImageSlide
+    imageDark={ImageB}
+    imageLight={ImageBLight}
+    key={1}
+    subtitle='Review your available and locked assets in your portfolio.'
+    title='View your assets in one place'
+  />,
+  <ImageSlide
+    imageDark={ImageC}
+    imageLight={ImageCLight}
+    key={2}
+    subtitle='Trade on the DEX and earn rewards from liquidity mining with crypto and dTokens.'
+    title='Maximize earning potential'
+  />,
+
+  <ImageSlide
+    imageDark={ImageD}
+    imageLight={ImageDLight}
+    key={3}
+    subtitle='Access financial opportunities with dTokens minted through decentralized vaults.'
+    title='Decentralized loans'
+  />]
+
+// Needs for it to work on web. Otherwise, it takes full window size
+const { width } = Platform.OS === 'web' ? { width: '375px' } : Dimensions.get('window')
+
+export function ImageSlide ({ imageDark, imageLight, title, subtitle }: CarouselImage): JSX.Element {
+  const { isLight } = useThemeContext()
+  return (
+    <View style={tailwind('flex-1 items-center justify-center py-8 px-10')}>
+      <Image
+        source={isLight ? imageLight : imageDark}
+        style={{ width: 220, height: 136 }}
+      />
+      <View style={tailwind('h-2/6 items-center justify-center')}>
+        <ThemedText
+          style={tailwind('text-xl font-semibold-v2 text-center mt-8')}
+          dark={tailwind('text-mono-dark-v2-900')}
+          light={tailwind('text-mono-light-v2-900')}
+        >
+          {translate('screens/OnboardingCarousel', title)}
+        </ThemedText>
+        <ThemedText
+          dark={tailwind('text-mono-dark-v2-900')}
+          light={tailwind('text-mono-light-v2-900')}
+          style={tailwind('font-normal-v2 text-center mt-2 mb-8')}
+        >
+          {translate('screens/OnboardingCarousel', subtitle)}
+        </ThemedText>
+      </View>
+    </View>
+  )
+}
+
+export function OnboardingCarouselV2 (): JSX.Element {
+  const { isLight } = useThemeContext()
+  return (
+    <SwiperFlatList
+      autoplay
+      autoplayDelay={30}
+      autoplayLoop
+      autoplayLoopKeepAnimation
+      data={slides}
+      index={0}
+      paginationActiveColor={isLight ? getColor('mono-dark-v2-100') : getColor('mono-dark-v2-900')}
```

Comment:
to be consistent with the design system
```suggestion
      paginationActiveColor={isLight ? getColor('mono-dark-v2-900') : getColor('mono-dark-v2-900')}
```